refellips/structureSE.py

```diff
@@ -346,6 +346,86 @@ def complex(self, wavelength):
         return real + 1j * 0.0
 
 
+class Sellmeier(ScattererSE):
+    r"""
+    Dispersion curves for Sellmeier oscillators.
+
+    Parameters
+    ----------
+    Am: {float, Parameter, sequence}
+        Amplitude of Sellmeier
+    En: {float, Parameter, sequence}
+        Center energy of oscillator
+    P: {float, Parameter, sequence}
+        Position of a pole (eV)
+    Einf: {float, Parameter}
+        Offset term
+    wavelength : float
+        default wavelength for calculation (nm)
+    name : str, optional
+        Name of material.
+
+    Notes
+    -----
+    Calculates dispersion curves for *k* oscillators, as implemented in CompleteEase.
```

Comment:
reference for CompleteEase implementation

Reply:
Also we're not doing *k* oscillators, this implementation is only 1. Can you use Gauss/Lorentz to see how to implement k oscillators (assuming you have the eqns from CompleteEase)?

Reply:
If Sellmeier only does 1 oscillator then remove the *k*